inserts/.gitignore

```diff
@@ -1,7 +1,6 @@
 
 # Lamby
 /.aws-sam
-/.lamby
```

Comment:
We do not need the `.lamby` directory anymore. The reason this was added is that without it, a deploy from you local machine means your development dependencies are hosed since vendor/bundle was for deployment. We now have scripting in place to warn about this and handle it automatically. So as we lean to good CI/CD practices... the exception will be dev host deploys with warnings and caveats.